Structure_oM/Loads/GeometricalLineLoad.cs

```diff
@@ -24,11 +24,12 @@
 using BH.oM.Geometry;
 using System.ComponentModel;
 using BH.oM.Quantities.Attributes;
+using BH.oM.Structure.Elements;
 
 namespace BH.oM.Structure.Loads
 {
     [Description("Distributed load to be applied over a line.")]
-    public class GeometricalLineLoad : BHoMObject, ILoad
+    public class GeometricalLineLoad : BHoMObject, ILoad, IElementLoad<IAreaElement> 
```

Comment:
```suggestion
    public class GeometricalLineLoad : BHoMObject, IElementLoad<IAreaElement> 
```
`IElementLoad<IAreaElement>` already inherits `ILoad`